### ISSUE TITLE
Failing Standard H91: "Using HTML form controls and links" when using icon button

### ISSUE THREAD

**Issue by mattdell** [DeloitteDigitalUK/common-accessible-components#7]
A button tag requires a name, which according to WCAG Standard H91 can be derived from either the text inside `<button>` element or 'title' attribute.

Since our icon button inserts a `<span>` tag into our `<button>` element a machine reader would not be able to derive a name without using a title attribute.

<img width="328" alt="screen shot 2016-02-14 at 14 33 47" src="https://cloud.githubusercontent.com/assets/2536442/13034261/92d17c46-d328-11e5-95eb-8660fd1f8252.png">

**Comment by mattdell** [comment]
I have a pull request for this @vitkon but I don't seem to have access to push a new branch to this repo.

<img width="478" alt="screen shot 2016-02-14 at 14 44 56" src="https://cloud.githubusercontent.com/assets/2536442/13034297/a6fc2562-d329-11e5-978c-8f65888f75d4.png">

**Comment by vitkon** [comment]
Your access should be fixed now, please add a PR
